### DISCUSSION TITLE
Is there a plugin for Now Playing export to HTML or something?

### DISCUSSION THREAD

**Discussion by Psilovybin** [audacious-media-player/audacious#108] [Q&A]
In winamp in windows, i have an export of Songtitle and song group to let my viewers know what's playing.  Would be nice to have something like what's in the default interface at the bottom exported to something like that with album art, title and group.

**Comment by radioactiveman** [comment]
I am not aware of such plugin for Audacious.

You could use the command line utility [`audtool`](https://manpages.org/audtool) (included with Audacious) to query the current track info. And then embed it into an HTML file. If you have enabled the MPRIS plugin in Audacious, you may also use [playerctl](https://github.com/altdesktop/playerctl). It can query even more metadata (including album art) with `playerctl metadata`.

**Comment by wildstar84** [comment]
I'm assuming you want to grab this data in real-time as the song-title changes, then write it (w/HTML wrapping) somewhere.  If so, I believe the answer I gave this guy awhile back may lead to a fix for you.  This is based on using Audacious's "Song Change" general plugin.  https://redmine.audacious-media-player.org/boards/2/topics/2821

Hope this helps,
Jim

**Reply by SephirothWS** [reply]
I am attempting to follow this reply on Bazzite, and for some reason I can't get the Song Change general plugin to write to file or even a named pipe like you suggested. The echo part works, but when I add in the > /path/to/file it just does nothing at all.

My intention is to simply write "%n" to a text file on song change, overwriting said file each time, so that OBS Studio can use that text document to display the title on screen

**Reply by radioactiveman** [reply]
What is your command? `echo "%n" > /tmp/file.txt` works fine here.

**Reply by SephirothWS** [reply]
`echo "%n" > /home/sephiroth/nowplaying.txt`

I am running a flatpak version of audacious. according to system monitor, the app is running under my user, so should have access to that file...

WHELP... I figured out my issue, had to go into the flatpak permissions editor and actually give the app permission to that directory... No wonder it wasn't working...
